cron/rotateIndexes.py

```diff
@@ -160,12 +170,16 @@ def initConfig():
         '20,0,0',
         options.configfile).split(',')
         )
-
     options.weekly_rotation_indices = list(getConfig(
         'weekly_rotation_indices',
         'events',
         options.configfile).split(',')
         )
+    options.previous_alias = list(getConfig(
```

Comment:
Is this expected to be a list? Seems like it would be a single value?

Reply:
Fixed! Thank you